scrapers/covid-19/govScrapers/getUK.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios');
+const logger = require('../../../utils/logger');
+
+const ukData = async () => {
+	try {
+		var data = { };
+		// eslint-disable-next-line max-len
+		const response = (await axios.get('https://api.coronavirus-staging.data.gov.uk/v1/data?filters=areaName=United%20Kingdom;areaType=overview&structure={"date": "date","todayTests": "newTestsByPublishDate","tests": "cumTestsByPublishDate","testCapacity": "plannedCapacityByPublishDate","newCases": "newCasesByPublishDate","cases": "cumCasesByPublishDate","hospitalized": "hospitalCases","usedVentilationBeds": "covidOccupiedMVBeds","newAdmissions": "newAdmissions","admissions": "cumAdmissions","newDeaths": "newDeathsByPublishDate","deaths": "cumDeathsByPublishDate"}')).data;
```

Comment:
no need to make this so long, just use a few extra lines to make it more readable in case we need to debug

Reply:
> no need to make this so long, just use a few extra lines to make it more readable in case we need to debug

wdym

Reply:
The length of this single line is so long, just split it up and make it more readable.

Reply:
i dont see the point in splitting up a url but alright

Reply:
https://github.com/disease-sh/API/pull/766/commits/9ab1ab1381e301b5e877a79f6bd3547d986eff37#diff-7cfcd3465d06aceda967694c28933cbeR7-R20 

made it so we can adapt the structure if they add/remove stuff